src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.targets

```diff
@@ -47,7 +47,6 @@ Copyright (C) 2012 Xamarin Inc. All rights reserved.
     <TargetFrameworkVersion Condition="'$(TargetFrameworkVersion)' == '' And '$(AndroidUseLatestPlatformSdk)' == 'False'">v5.0</TargetFrameworkVersion>
     <LangVersion Condition=" '$(LangVersion)' == '' ">8.0</LangVersion>
     <AndroidClassParser Condition=" '$(AndroidClassParser)' == '' ">jar2xml</AndroidClassParser>
-    <AndroidCodegenTarget Condition=" '$(AndroidCodegenTarget)' == '' ">XAJavaInterop1</AndroidCodegenTarget>
```

Comment:
This should already be set in `Microsoft.Android.Sdk.DefaultProperties.targets`.